src/main/java/com/redislabs/redisai/DagRunCommands.java

```diff
@@ -0,0 +1,11 @@
+package com.redislabs.redisai;
+
+interface DagRunCommands {
```

Comment:
If we need a chain call. We can change interface like this:
```
package com.redislabs.redisai;

interface DagRunCommands<T> {
  T setTensor(String key, Tensor tensor);

  T getTensor(String key);

  T runModel(String key, String[] inputs, String[] outputs);

  T runScript(String key, String function, String[] inputs, String[] outputs);
}
```
`public class Dag implements DagRunCommands<Dag>` 

WDYT?  : )

Reply:
agree :+1: . applying changes